simglucose/simulation/env.py

```diff
@@ -34,7 +36,58 @@ def risk_diff(BG_last_hour):
 
 
 class T1DSimEnv(object):
+    '''
+    Observation:
+        Type: Box(4)
+        Num     Observation                 Min         Max
+        0       BG                          69.         351.
+        1       Insulin                     0.0         Inf
+        2       CHO                         0.0         Inf
+        3       CGM                         0.0         Inf
```

Comment:
In fact, CGM is the observation. It is ranged from 70 to 350 in the gym environment.